example/plugin/toerror/Readme.md

```diff
@@ -0,0 +1,25 @@
+The deriveToError function is useful for composing bool-returning functions along with traditional error-returning functions.
+
+Given the following input pseudo-code:
+
+```go
+expectKey := func(k some_key) (some_value, bool) {
+    value, keyExists = some_map[k]
+    return value, keyExists
+}
+transformed := deriveToError(fmt.Errorf("false returned. something's wrong!"), expectKey)
+```
+
+goderive will generate the following code:
+
+```go
+func deriveToError(err error, f func(i some_key) (a some_value, b bool)) func(i some_key) (some_value, error) {
```

Comment:
Something is not right, the code generation should replace the code in the readme with the actual generated code.  Did you run `make` in the example folder?

Reply:
regenerated by make. resolving it.